e2e-test/src/tests/delete.test.js

```diff
@@ -23,12 +24,13 @@ describe('delete dataset', () => {
     await page.waitForXPath(vListTile({ title: 'deleteTest.csv ' }));
     await page.waitForXPath(vListTile({ title: 'deleteTest.csv ', content: 'Dataset ready for analysis' }));
 
+    await new Promise((resolve) => setTimeout(resolve, 1000));
```

Comment:
Wait for data set to be uploaded

Reply:
If you feel like you need to explain yourself in the PR, you probably also need to explain yourself to anyone reading the code. I think it's generally a good idea to include a comment explaining why a sleep is necessary.